src/ashley/locale/fr_FR/LC_MESSAGES/django.po

```diff
@@ -258,52 +258,52 @@ msgid "Post topic as"
 msgstr "Publier la discussion comme"
 
 msgid "Poll question"
-msgstr ""
+msgstr "Question du sondage"
 
 msgid "Enter a question to associate a poll with the topic or leave blank to not create a poll."
 msgstr "Entrez une question pour associer un sondage à la discussion, ou laissez le champ vide pour ne pas créer de sondage."
 
 msgid "Maximum number of poll options per user"
-msgstr ""
+msgstr "Nombre maximum de choix par utilisateur"
 
 msgid "This is the number of options each user may select when voting."
-msgstr ""
+msgstr "Il s'agit du nombre de choix que chaque utilisateur peut sélectionner lorsqu'il participe au sondage."
 
 msgid "For how many days the poll should be run?"
-msgstr ""
+msgstr "Indiquez la durée du sondage (nombre de jours)"
 
 msgid "Enter 0 or leave blank for a never ending poll."
-msgstr ""
+msgstr "Entrez 0 ou laisser le champ vide si vous souhaitez que le sondage soit permanent."
 
 msgid "Allow re-voting?"
-msgstr ""
+msgstr "Autoriser le changement de vote ?"
```

Comment:
Changement de vote ou double vote ?

Reply:
C'est pour changer de vote.
Le texte d'aide associé à ce champ de formulaire est `If enabled users are able to change their vote.`

Reply:
Ok then.